lib/livebook/session.ex

```diff
@@ -892,6 +892,7 @@ defmodule Livebook.Session do
   @spec default_notebook() :: Notebook.t()
   def default_notebook() do
     %{Notebook.new() | sections: [%{Section.new() | cells: [Cell.new(:code)]}]}
+    |> Notebook.set_default_hub()
```

Comment:
I would push it down the line and set it directly here:

https://github.com/livebook-dev/livebook/blob/893b82ac8344a4109d1bdd3eee36013e7e9e79fe/lib/livebook/notebook.ex#L111

We already use an arbitrary id there (personal), so using the default makes sense.

Reply:
That was my first approach, but I got an error related to Livebook.Storage. Would it be possible to call default_hub there? How?

Reply:
Ah that's because we import notebooks for the learn section at compile time and it calls `Notebook.new`, but the `Storage` process is not started.

Actually, I looked at places where we call `Notebook.new` and I think it be agnostic to any configuration, so setting the id here sounds good. I would just inline `hub_id: Livebook.Hubs.get_default_hub().id` here :)